src/env.ts

```diff
@@ -0,0 +1,79 @@
+import { createEnv } from "@t3-oss/env-nextjs";
```

Comment:
@arunanshub I like this approach because it is setting strong validation for environment variables. Given we don't have good test cases, I hope you have verified the changes locally for possible impact